easybuild/easyconfigs/p/Pyomo/Pyomo-5.5.0-foss-2018b-Python-3.6.6.eb

```diff
@@ -0,0 +1,51 @@
+easyblock = 'Bundle'
+
+name = 'Pyomo'
+version = '5.5.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.pyomo.org/'
+description = """ Pyomo is a Python-based open-source software package that supports a diverse set of optimization
+ capabilities for formulating and analyzing optimization models. """
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+dependencies = [('Python', '3.6.6')]
+
+exts_defaultclass = 'PythonPackage'
+
+exts_default_options = {
+    'download_dep_fail': True,
+    'use_pip': True,
+}
```

Comment:
Not needed with `PythonBundle` except `use_pip = True`

Reply:
@vanzod Done